docs/B-Computations/a-simple-calculation.md

```diff
@@ -184,50 +194,50 @@ C compilers reject statements such as:
 Adding the statements to store the area in memory yields:
 
 ```c
-#include <stdio.h>                   // for printf, scanf
+#include <stdio.h>               // for printf, scanf
 
- int main(void)
- {
-         const float pi = 3.14159f;   // pi is a constant float 
-         float radius;                // radius is a float
-         float area;                  // area is a float
+int main(void)
+{
+    const float pi = 3.14159f;   // pi is a constant float 
+    float radius;                // radius is a float
+    float area;                  // area is a float
 
-         printf("Enter radius : ");   // prompt user for radius input
-         scanf("%f", &radius);        // accept radius value from user 
+    printf("Enter radius : ");   // prompt user for radius input
+    scanf("%f", &radius);        // accept radius value from user 
 
-         area = pi * radius * radius; // calculate area from radius
+    area = pi * radius * radius; // calculate area from radius
 
-         // ... completed below
+    // ... completed below
 
-         return 0;
- }
+    return 0;
+}
 ```
 
-Since the C language does not define an exponentiation operator, we need to calculate the square of the radius explicitly. Later, we will learn the pow procedure to perform exponentiation.
+Since the C language does not define an exponentiation operator, we need to calculate the square of the radius explicitly. Later, we will learn the `pow` procedure to perform exponentiation.
 
 ## Simple Output
 
-The **`printf(...)`** instruction reports the value of a variable or expression to the user \(that is, copies the value to the standard output device\).  The instruction takes the form:
+The `printf(...)` instruction reports the value of a variable or expression to the user \(that is, copies the value to the standard output device\).  The instruction takes the form:
 
 ```c
  printf(format, expression);
 ```
 
-This statement calls the **`printf()`** procedure, which performs the operation.  We say that format and expression are arguments in our call to **`printf()`**. 
+This statement calls the `printf()` procedure, which performs the operation.  We say that _**format**_ and _**expression**_ are arguments in our call to `printf()`. 
 
 ### Format
 
-_**format**_ is a string literal describing how to convert data stored in memory into text readable by the user.  _**format**_ contains the conversion specifier and any characters to be output directly.  The conversion specifier begins with a % symbol and identifies the type of the source variable.  The most common specifiers are listed below.
+_**format**_ is a string literal describing how to convert data stored in memory into text readable by the user.  _**format**_ contains the conversion specifier and any characters to be output directly.  The conversion specifier begins with a **`%`** symbol and identifies the type of the source variable.  The most common specifiers are listed below.
```

Comment:
Since you already wrapped the `%` in backticks, I think there is no need to add emphasis with `**`